pkg/scheduler/test_scheduler.go

```diff
@@ -14,12 +14,15 @@ import (
 	"github.com/kubeshop/testkube/internal/common"
 	"github.com/kubeshop/testkube/pkg/api/v1/testkube"
 	"github.com/kubeshop/testkube/pkg/executor/client"
+	"github.com/kubeshop/testkube/pkg/executor/content"
 	testsmapper "github.com/kubeshop/testkube/pkg/mapper/tests"
 	"github.com/kubeshop/testkube/pkg/workerpool"
 )
 
 const (
 	containerType = "container"
+	// tempDir is the temporary directory used to evaluate a type of a 'git' source
+	tempDir = "/tmp"
```

Comment:
you can use it like I did for git check

			dir, err := os.MkdirTemp("", "checkout")
			if err != nil {
				return s.Error(c, http.StatusBadGateway, err)
			}
			defer os.RemoveAll(dir) // clean up

Reply:
Or os.GetTemp()